### DISCUSSION TITLE
Failing to parse openning XML declaration on W10

### DISCUSSION THREAD

**Discussion by footpeg** [FuLoRi/Analyze-LastPassVaultGUI#17] [Q&A] [answered]
Using Chrome on Windows 10 and the GUI version all seems to function AOK except the nanlysis fails on parsing the openning XML declaration. So no analysis completes

The error reported is
Cannot convert value "<?xml version="1.0" encoding="UTF-8"?>

**Comment by FuLoRi** [comment]
@footpeg: The error message seems to have been truncated. Any chance you could try pasting it again? Thanks!

**Reply by footpeg** [reply]
Oops, apologies missed that

Cannot convert value "<?xml version="1.0" encoding="UTF-8"?>

It continues thru the paste (displaying it in red) and writes a blank file. saying it completed

**Reply by footpeg** [reply]
poop!  it dropped the chars when I hit reply

Cannot convert value "<?xml version="1.0" encoding="UTF-8"?>"
<img width="785" alt="error" src="https://user-images.githubusercontent.com/14120096/212802157-740d9ce0-40bc-4ff6-9007-a123392b6b73.PNG">

**Reply by FuLoRi** [reply]
Apologies for the delayed response. Life happened and this sort of slipped through the cracks.

Assuming you're still having this issue, any chance you could post a screenshot of the end of the error message? It may shed more light on the problem.

Thanks!

**Comment by footpeg** [comment]
Rob,


No need for apologies. Life does happen and to be honest I'd forgotten
about this - in the normal course of my life

Attached is more detail, Powershell version, and some of the process. There
are two PS1 screenshots. One from the beginning of the output and one from
the end. In between was just more of the red reproduction of the input

Pete

On Tue, 7 Feb 2023 at 13:57, Rob Woodruff ***@***.***> wrote:

> Apologies for the delayed response. Life happened and this sort of slipped
> through the cracks.
>
> Assuming you're still having this issue, any chance you could post a
> screenshot of the end of the error message? It may shed more light on the
> problem.
>
> Thanks!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/FuLoRi/Analyze-LastPassVaultGUI/discussions/17#discussioncomment-4888970>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADLXJIHX2UYA4JAHVAXVNMLWWG2X7ANCNFSM6AAAAAAT3ZZDNU>
> .
> You are receiving this because you were mentioned.Message ID:
> <FuLoRi/Analyze-LastPassVaultGUI/repo-discussions/17/comments/4888970@
> github.com>
>


-- 
==========================
Peter Sandilands

Sandilands Security Pty Ltd
0439 738 371
***@***.***
==========================

**Reply by FuLoRi** [reply]
Hi Pete,

The attachments didn't come through. If you don't mind, please go [here](https://github.com/FuLoRi/Analyze-LastPassVaultGUI/discussions/17) and attach them.

Thanks!

-Rob

**Comment by footpeg** [comment]
[screenshots.docx](https://github.com/FuLoRi/Analyze-LastPassVaultGUI/files/10679793/screenshots.docx)
Cripes, anybody would think I don't know how to computer.

**Reply by FuLoRi** [reply, marked as answer]
😆 It's all good. Thanks for the screenshots.

Looking at the second one showing "Analysis complete", you can see in the Paste field that the XML is truncated. I'm almost certain that's why you're getting the error stating that there is an unclosed literal string.

The chrome extension that DevTools is having trouble with is the LastPass extension.  It could be a coincidence and may have nothing to do with why the XML is being truncated.

I'd be curious to know whether the XML is truncated if you copy it out of the DevTools screen and paste it into Notepad. If it's not truncated there, then you can save it as a .XML file and use the Browse option instead of the Paste option to import it into my script for analysis.

If it's still truncated, though, then you might try disabling the LastPass extension or trying it from an Incognito window to rule out any interference from misbehaving extensions.
